app/scripts/components/exploration/components/timeline/timeline-controls.tsx

```diff
@@ -68,18 +68,20 @@ interface TimelineControlsProps {
 }
 
 
-export function useInitialScale(width) {
-  return useMemo(() => {
-    const now = new Date();
-    return scaleTime()
-      .domain([startOfYear(now), endOfYear(now)])
-      .range([0, width]);
-  }, [width]);
+export function getInitialScale(width) {
+  const now = new Date();
+  return scaleTime()
+    .domain([startOfYear(now), endOfYear(now)])
+    .range([0, width]);
 }
 
 export function TimelineDateAxis(props: Omit<TimelineControlsProps, "onZoom">) {
   const { xScaled, width } = props;
-  const initialScale = useInitialScale(width);
+
+  const initialScale = useMemo(() => {
+    return getInitialScale(width);
+  }, [width]);
```

Comment:
⛏️ : I would simplify like this 

`const initialScale = useMemo(() => getInitialScale(width), [width]);`

not a blocker though